### DISCUSSION TITLE
Custom changelog

### DISCUSSION THREAD

**Discussion by revolunet** [semantic-release/semantic-release#1978] [Q&A]
Hi there,

I'm looking for guidance on how to customize my changelog and release notes content when using semantic-release and GitHub. I'd like to customize the content of these on each new release, by adding some markdown based on the "diff" since previous release. Ideally this would happen only on some specific commit messages, defaulting to the default in other cases.

Thank you !

**Comment by kyranjamie** [comment]
I'd second this. The output from the release-notes-generator isn't that great, to be honest, when compared to tools like changesets.

With a little bit of modification—using the commit body, some reformatting—we could generate much nicer changelogs.
